laze-project.yml

```diff
@@ -856,6 +856,19 @@ modules:
         FEATURES:
           - ariel-os/coap-server
 
+  - name: coap-server-config-storage
+    help: Configure the CoAP server to accept requests depending on build- and runtime configuration
+    depends:
+      - coap
+      - sw/storage
+    provides_unique: [coap-server-config]
+    env:
+      global:
+        FEATURES:
+          - ariel-os/coap-server-config-storage
+        CARGO_ENV:
+          - PEERS_YML=$\\${PWD}/peers.yml
```

Comment:
```suggestion
          - PEERS_YML=${appdir}/peers.yml
```

Reply:
This will be obsolete when using a laze variable. (It may apply to the laze variable default, though).